### PR TITLE
updated to match drone1 yaml format

### PR DESCRIPTION
**Pull request by cloudnautique**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,129 +1,200 @@
...
-      branch: [master, release/*]
-      event: tag
+    tag: "${DRONE_TAG}"
+    username:
+      from_secret: docker_username
+  when:
+    event:
+    - tag
+    ref:
+    - refs/head/master
```
Comment (jgreat) [inline .drone.yml]: Is it necessary to have the ref: block here.  At least with drone 0.8 Tags only have the proper ref if the commit tagged it he current head of the branch.

File: .drone.yml [hunk 2]
```diff
@@ -1,129 +1,200 @@
...
+  pull: default
+  image: rancher/dapper:v0.4.1
+  commands:
+  - dapper chart/copy
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+  when:
+    event:
+    - promote
```
Comment (jgreat) [inline .drone.yml]: shouldn't this be a `deployment` event?

File: .drone.yml [hunk 3]
```diff
@@ -1,129 +1,200 @@
...
-      - source: google_auth_key
-        target: GOOGLE_CREDENTIALS
-    when:
-      event: [ tag, deployment ]
+    token:
+      from_secret: google_auth_key
+  when:
+    event:
+    - tag
+    - promote
```
Comment (jgreat) [inline .drone.yml]: shouldn't this be a `deployment` event?

File: .drone.yml (hunk 2 above)
Comment (cloudnautique) [reply .drone.yml]: deployment has been deprecated in favor of the less confusing promote. Based on docs.

**Pull request merged**